guevaraTorres/assignments/twopointers/lc1/lc1.js

```diff
@@ -3,8 +3,59 @@
 
 // Write a function to return the indices of the two numbers (i.e. the pair) such that they add up to the given target.
 
+/**
+ * 
```

Comment:
Please note that this solution works under the assumption that the given array is sorted.